packages/fxa-settings/src/pages/Signin/Signin/mocks.tsx

```diff
@@ -0,0 +1,5 @@
+import { ReactComponent as PocketIcon } from './pocket.svg';
```

Comment:
Is just a mock, not sure if we want to add copyright header or no.

```js
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Reply:
Same! 🤷

Reply:
I wish I knew where I was originally pointed to a couple of years ago but FWIW, last I checked the license is supposed to be listed at the top of _every_ file, but if we can't add one because the code doesn't allow comments, then we're supposed to add a license file in directory the file is located in.

Reply:
Filed https://mozilla-hub.atlassian.net/browse/FXA-6700.

```sh
# cd packages
git grep -L "This Source Code Form is subject to the terms" -- ':!fortress/**' ':!123done/**' '*.js' '*.jsx' '*.ts' '*.tsx'
```

Currently about 339 files w/o headers (or 230 if we also ignore `':!*.stories.*' ':!*.test.*' ':!*.spec.*'` files).

Reply:
@pdehaan that's so many files 🫣 What if we exclude `.ftl` files?

Reply:
Oh, I'm only grepping the `*.js`, `*.jsx`, `*.ts`, and `*.tsx` files.

```sh
node copyright.mjs
# .js => 64
# .ts => 148
# .tsx => 127
# { sum: 339 }
```

And if we ignore the `*.stories.*`, `*.test.*`, and `*.spec.*` files, that drops to:
```sh
node copyright.mjs
# .js => 61
# .ts => 109
# .tsx => 60
# { sum: 230 }
```

<details>
<summary>clicko expando</summary>

```js
// copyright.mjs
import fs from "node:fs/promises";
import path from "node:path";

const log = await fs.readFile("./copyright.log", "utf-8");

const lines = log.trim().split("\n");

const files = lines.reduce((acc, line) => {
  const ext = path.extname(line);
  if (!acc[ext]) {
    acc[ext] = [];
  }
  acc[ext].push(line);
  return acc;
}, {});

let sum = 0;
Object.entries(files)
  .forEach(([ext, files]) => {
    sum += files.length;
    console.log(`${ext} => ${files.length}`);
  });

console.log({ sum });
```
</details>